config/routes.rb

```diff
@@ -14,6 +14,10 @@
     end
   end
 
+  namespace :admin do
+    get '/dashboard', to: "admins#show"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.